libnavigation-base/src/main/java/com/mapbox/navigation/base/options/EHorizonOptions.kt

```diff
@@ -113,8 +123,15 @@ class EHorizonOptions private constructor(
                 length = length,
                 expansion = expansion,
                 branchLength = branchLength,
-                includeGeometries = includeGeometries
+                minTimeDeltaBetweenUpdates = minTimeDeltaBetweenUpdates
             )
         }
+
+        private companion object {
+            private const val DEFAULT_LENGTH = 500.0
+            private const val DEFAULT_EXPANSION = 0
+            private const val DEFAULT_BRANCH_LENGTH = 50.0
+            private val DEFAULT_DELTA = null
```

Comment:
```suggestion
            internal const val DEFAULT_LENGTH = 500.0
            internal const val DEFAULT_EXPANSION = 0
            internal const val DEFAULT_BRANCH_LENGTH = 50.0
            internal val DEFAULT_DELTA = null
```
so that they can be reused in tests

Reply:
fixed